### PR TITLE
🏗️ Build spider: Central Valley School District Board

### PR DESCRIPTION
**Pull request by SimmonsRitchie**
## What's this PR do?
Adds a spider to scrape meetings from the boarddocs website for Central Valley School District Board. The new spider is called `spo_cvsd`.

## Why are we doing this?
Requested by City Bureau's partners in Spokane.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl spo_cvsd -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://go.boarddocs.com/wa/cvsd/board.nsf/public).

## Are there any smells or added technical debt to note? 
- Fixture set up in the testing suite for this spider is a little more complicated because we need to ensure `meta` is available in response. A comment has been left in the file to provide clarity.
- Just a note: `scrapy validate spo_cvsd` passes with 100% for all items for me.
- Not a big deal but the branch name for this repo is misspelled. Should be "build-cvsd". Just noting in case the reviewer fetches the branch and is confused.

### CODE REVIEW

File: tests/test_spo_cvsd.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    join(dirname(__file__), "files", "spo_cvsd.json"),
+    url="https://go.boarddocs.com/wa/cvsd/Board.nsf/BD-GetMeetingsList?open&0.320616755508751",  # noqa: E501
+)
+start_date = datetime(2024, 1, 22)
+request_with_meta = Request(
+    url="https://go.boarddocs.com/wa/cvsd/Board.nsf/BD-GetMeeting?open&0.934001628042688",  # noqa: E501
+    meta={"start_date": start_date, "meeting_id": "CYXL4V54CB3F"},
+)
+response_body = open(
+    join(dirname(__file__), "files", "spo_cvsd_detail.json"), "rb"
```
Comment (LienDang) [inline tests/test_spo_cvsd.py]: Minor thing: should we save the details content in .html file instead of .json?

**Review by LienDang — APPROVED** [review]
LGTM

File: tests/test_spo_cvsd.py (hunk 1 above)
Comment (SimmonsRitchie) [reply tests/test_spo_cvsd.py]: Good catch, @LienDang. Yes, this should be an `html` extension. It's now fixed. Thank you for spotting that 🙏

**Comment by SimmonsRitchie** [comment]
Thank you, @LienDang. I've fixed the html/json fixture issue. I also noticed an issue with how I used freezegun in that test. That's fixed too. Merging now 🚀

**Pull request merged**